src/Forecast.js

```diff
@@ -0,0 +1,61 @@
+import { Grid, Box, Dialog, DialogTitle } from "@mui/material";
+import {
+  LineChart,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Line,
+  Tooltip,
+} from "recharts";
+
+export default function Forecast(props) {
+  const { forecastData } = props;
+  if (!("city" in forecastData)) {
+    return <div></div>;
```

Comment:
Why bother returning an empty div like such? I think `return null` would be more appropriate if you do not want to render anything